vignettes/working-with-canvas-data.Rmd

```diff
@@ -0,0 +1,97 @@
+
+After reading in the data, the data can be used to calculate grades as in the case study vignettes. 
+
+Here is an example of the transformation Gradebook applies to Canvas grade data.
+
+```{r, echo = FALSE}
+  canvas_demo_data <- readr::read_csv(system.file("extdata", "canvas_demo_data.csv", package = "gradebook"), trim_ws = FALSE)
+  knitr::kable(canvas_demo_data)
+```
+
+Here is the processed dataframe.
+
+```{r echo = FALSE}
+  proc_canvas_data <- read_files(system.file("extdata", "canvas_demo_data.csv", package = "gradebook"))
+  knitr::kable(proc_canvas_data)
+```
+
+
+
+## Limitations
+
+There are limitations to using assignment data sourced from Canvas, resulting in a reduction of possible grading policies in Gradebook. The primary limitation surrounds applies lateness policies when using data sourced from Canvas. 
+
+The grades csv exported from Canvas does not include information on assignment lateness. As mentioned below, Canvas also can export files detailing late assignments. However, this lateness file omits the time that the assignment was due, making it not possible to accurately calculate how late any late assignment is. 
+
+As a result, we do not allow lateness policies to be used when the data is sourced from Canvas. Attempts to do so will raise an error. 
+
+Currently, our recommendation is for Canvas users to apply any late policies in Canvas, since Canvas can apply simple lateness policies. 
+
+## Other Data
+
+In addition to the grades file, Canvas also exports information that can be relevant to grade calculation. Examples include the lateness csv file and a roster csv file. 
+
+The lateness file details which assignments where submitted late by which students. Due to the drawbacks mentioned above, we are currently not using this file to calculate late policies in Gradebook. 
+
+The roster csv file gives all student names and their contact information, which can be relevant if instructors desire to have student contact information available in the exported grades file. Integrating roster information is currently not supported.
+
+Both of these files are accessed through the "New Analytics" tab available on the right side of the course homepage. To download these files, click Analytics on the home page, navigate to the Reports tab and download the desired report. Reference the photos below. 
+
+```{r echo=FALSE, out.width='100%'}
+  knitr::include_graphics(system.file("extdata", "canvasAnalytics.png", package = "gradebook"))
+
+```
+
+```{r echo=FALSE, out.width='100%'}
+  knitr::include_graphics(system.file("extdata", "canvasAnalyticsOptions.png", package = "gradebook"))
+
+```
+
+
+Currently, providing files other than the grades file is not supported by Gradebook. However, we have built the infrastructure to read in these files, should the need or opportunity arise. 
+
+For full information on the formatting of Canvas exported data, see the "Detailed Canvas Export Info" vignette. 
```

Comment:
Same thing as the last vignette comment, I'll PR into Zach's repo to suggest any minor changes